src/writing-your-own-application/generate-project/esp-template.md

```diff
@@ -39,10 +39,10 @@ Before going further, let's see what these files are for.
 - [.gitignore][gitignore]
     - Tells `git` which folders and files to ignore
 - [Cargo.toml][cargo-toml]
-    - The usual Cargo manifest declaring some meta-data and dependencies of the project
+    - The usual Cargo manifest declares some meta-data and dependencies of the project
 - LICENSE-APACHE, LICENSE_MIT
     - Those are the most common licenses used in the Rust ecosystem
-    - If you want to apply a different license, you can delete these files and change the license in `Cargo.toml`
+    - If you want to apply for a different license, you can delete these files and change the license in `Cargo.toml`
```

Comment:
Do you need to apply to some institution to get a license for your open source project?

If not, then suggest sticking with "apply a different license" or "use a different license".